### ISSUE TITLE
Issue with vuelidate

### ISSUE THREAD

**Issue by AlexanderSDavidson** [alexjoverm/v-runtime-template#15]
After installing vuelidate v-runtime-template now throws an error, they are in separate component.  

```
    app.js:44784 [Vue warn]: Error in render: "TypeError: Cannot read property 'validations' of undefined"

found in

---> <VRuntimeTemplate
```

Any ideas? Is this a build problem?

**Comment by Solinx** [comment]
The issue comes down to v-runtime not having $options set on it's dynamically generated component, while Vuelidate expects $options to be present. I used this to avoid the error:

`// Add a mock mixin to avoid issues between v-runtime-template and Vuelidate
Vue.mixin({
  data: function() {
    if (this && !this.$options) {
      this.$options = {};
    }
    return {}
  }
});`

**Comment by alexjoverm** [comment]
Fixed by 1.7.0
